schemas/1.6.0/adaptive-card.json

```diff
@@ -2219,6 +2225,20 @@
         }
       ]
     },
+    "InputStyle": {
+      "description": "Style hint for input.",
+      "anyOf": [
+        {
+          "enum": [
+            "readWrite",
```

Comment:
I don't think this is the right style name:
1. `default` would be "same as today" and inputs are definitely already read-write
2. Given that inputs are read-write in `default` mode, `readWrite` makes no sense. The style is purely visual, it doesn't make inputs read-only. We need a style name that acknowledges that. I suggest `borderLess`